plugins/ecs-bridge/commands/commands.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package commands
+
+import (
+	"github.com/aws/amazon-ecs-cni-plugins/plugins/ecs-bridge/engine"
+	"github.com/aws/amazon-ecs-cni-plugins/plugins/ecs-bridge/types"
+	log "github.com/cihub/seelog"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types/current"
+)
+
+// Add invokes the command to add ENI to a container's namespace
+func Add(args *skel.CmdArgs) error {
+	defer log.Flush()
+	return add(args, engine.New())
+}
+
+// Del invokes the command to remove ENI from a container's namespace
+func Del(args *skel.CmdArgs) error {
+	defer log.Flush()
+	return del(args, engine.New())
+}
+
+func add(args *skel.CmdArgs, engine engine.Engine) error {
+	conf, err := types.NewConf(args)
+	if err != nil {
+		// TODO: We log and return errors throughout this function.
+		// Either should be sufficient.
+		log.Errorf("Error loading config from args: %v", err)
+		return err
+	}
+
+	log.Infof("Creating the bridge: %s", conf.BridgeName)
+	bridge, err := engine.CreateBridge(conf.BridgeName, conf.MTU)
+	if err != nil {
```

Comment:
I'm not quite sure what you mean by this. Are you asking for better error messages or more logging?